### ISSUE TITLE
myradio group: ads

### ISSUE THREAD

**Issue by Daxy7** [uBlockOrigin/uAssets#18578]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/124740436/235392297-c4fc6290-73a6-4a70-96a9-3f6a7683f21d.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other content blocker extensions.
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

```
https://irishradiolive.com/
https://myonlineradio.at/
https://myonlineradio.de/
https://myonlineradio.hu/
https://myonlineradio.sk/
https://myradioendirect.fr/
https://myradioenvivo.ar/
https://myradioenvivo.mx/
https://myradioonline.cl/
https://myradioonline.it/
https://myradioonline.pl/
https://myradioonline.ro/
https://ukradiolive.com/
```

### Category

ads

### Description

Filter
```
irishradiolive.com,myonlineradio.at,myonlineradio.de,myonlineradio.hu,myonlineradio.sk,myradioendirect.fr,myradioenvivo.ar,myradioenvivo.mx,myradioonline.cl,myradioonline.it,myradioonline.pl,myradioonline.ro,ukradiolive.com##.bnr-300-250
```
to be added

### Other extensions used

none

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml

uBlock Origin 1.50.0:
 since: 5d.4h.23m
Firefox: 114
filterset (summary):
 network: 85108
 cosmetic: 42343
 scriptlet: 21411
 html: 1122
listset (total-discarded, last-updated):
 removed:
  urlhaus-1: null
  plowe-0: null
 added:
  https://gitlab.com/magnolia1234/bypass-paywalls-clean-filters/-/raw/5811c0757dad027282fb82f9c751c79fe4f9306e/bpc-paywall-filter.txt: 905-0, 1h.12m
  https://raw.githubusercontent.com/Daxy7/webSiteFilters/main/Ublock/getadmiral_bypass_warnings.txt: 1736-0, 1h.12m
  https://raw.githubusercontent.com/nuchi/gmail-inbox-high-contrast-restoration/master/gmail-high-contrast-restoration.txt: 61-0, 1h.12m
  ublock-annoyances: 5065-7, 1h.12m
 default:
  user-filters: 1868-13, never
  ublock-filters: 34431-80, 1h.12m
  ublock-badware: 7518-0, 1h.12m
  ublock-privacy: 447-0, 1h.12m
  ublock-unbreak: 2069-0, 1h.12m
  ublock-quick-fixes: 240-0, 1h.12m
  easylist: 66738-373, 1h.12m
  easyprivacy: 30598-1139, 1h.12m
filterset (user): [array of 839 redacted]
switchRuleset:
 added: [array of 5 redacted]
hostRuleset:
 added: [array of 3 redacted]
userSettings:
 advancedUserEnabled: true
 suspendUntilListsAreLoaded: false
hiddenSettings: [none]
supportStats:
 allReadyAfter: 725 ms (selfie)
 maxAssetCacheWait: 152 ms
 unsuspendAfter: 495 ms
popupPanel:
 blocked: 8
 network:
  doubleclick.net: 1
  facebook.net: 1
  googlesyndication.com: 4
  googletagmanager.com: 1
  quantcast.com: 1
 extended:
  ##ins.adsbygoogle
  ##.div-gpt-ad:not([style^="width: 1px; height: 1px; position: ab…
  ##[class^="div-gpt-ad"]:not([style^="width: 1px; height: 1px; po…
  ##[id^="div-gpt-ad"]:not([style^="width: 1px; height: 1px; posit…
  ##div[id^="div-gpt-"]:not([style^="width: 1px; height: 1px; posi…
  #@#[class^="div-gpt-ad"]
  #@#[id^="div-gpt-ad"]
  #@#div[id^="div-gpt-"]

**Comment by mapx-** [comment]
report at easylist or other regional repositories

**Comment by krystian3w** [comment]
Anti-adblock goes on that page:

https://github.com/uBlockOrigin/uAssets/assets/35370833/3d62d30d-1aad-4e1b-87ab-5239211a0e3d


I reproduced too on Hungary version but I didn't use daily list for that region:

https://github.com/uBlockOrigin/uAssets/assets/35370833/59a9a8fa-03e3-4711-b117-c6ac1c657c96

**Comment by krystian3w** [comment]
https://github.com/MajkiIT/polish-ads-filter/issues/22052

**Comment by Daxy7** [comment]
Unable to reproduce. How are you getting this popup message?

**Comment by krystian3w** [comment]
At first visit.

**Comment by garry-ut99** [comment]
> krystian3w : Anti-adblock goes on that page:  [SCREENSHOOT](https://github.com/uBlockOrigin/uAssets/assets/35370833/3d62d30d-1aad-4e1b-87ab-5239211a0e3d)

Yes, it's still here, anti-adblock at `myradioonline.pl`, on both Firefox/Opera, also, after closing the initial message, other anti-adblock messages are  still embed on page : [SCREENSHOOT 2](https://github.com/uBlockOrigin/uAssets/assets/72945564/b5d369b0-1b7c-47ba-9efc-aa823b1243d7), [SCREENSHOOT 3](https://github.com/uBlockOrigin/uAssets/assets/72945564/1447e748-5a62-41b4-bf67-63572f28f8fe)
Also an issue which is opened on POL repo, is not fixed ( https://github.com/MajkiIT/polish-ads-filter/issues/22052 )

**Comment by D4niloMR** [comment]
I can reproduce the anti-adblock only on `myradioonline.pl`
